transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -440,22 +441,22 @@ object TransactorService {
   }
   
   def journalBlockReferenceToEvent(ref: Reference): JournalEvent = {
-    JournalEvent().withEvent(Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
+    JournalEvent(JournalEvent.Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
```

Comment:
they are in a different package, so you can't accidentally shadow.
I was at a loss for a better name, as it really is just a JournalEvent translated.